.rubocop.yml

```diff
@@ -9,16 +9,17 @@ AllCops:
   DisplayCopNames: true
 
 Layout/LineLength:
-  Max: 160
+  Max: 120
 Metrics/MethodLength:
   Include: ["app/controllers/*"]
   Max: 20
 Metrics/AbcSize:
   Include: ["app/controllers/*"]
-  Max: 30
```

Comment:
I notice that you change these rules in the robocop file. Remember that these rules were created to reinforce good [practices](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/). We should try other approaches to solve this problem, like splitting the function in two or more. If the complaint is related to specifications required in the project, you can disable the rule in the source code in the line where the offense happens.